src/catalog/src/durable/objects.rs

```diff
@@ -695,65 +695,18 @@ pub struct Snapshot {
         BTreeMap<proto::ServerConfigurationKey, proto::ServerConfigurationValue>,
     pub default_privileges: BTreeMap<proto::DefaultPrivilegesKey, proto::DefaultPrivilegesValue>,
     pub system_privileges: BTreeMap<proto::SystemPrivilegesKey, proto::SystemPrivilegesValue>,
+    pub storage_metadata: BTreeMap<proto::StorageMetadataKey, proto::StorageMetadataValue>,
+    pub unfinalized_shards: BTreeMap<proto::UnfinalizedShardKey, ()>,
+    pub persist_txn_shard: BTreeMap<(), proto::PersistTxnShardValue>,
```

Comment:
What's the reason for storing this as a value instead of a key? Is there only ever a single value?

Reply:
Yes, there is only ever a single value.